analysis_options.yaml

```diff
@@ -6,6 +6,8 @@ analyzer:
     strict-casts: true
     strict-inference: true
     strict-raw-types: true
+  exclude:
+    - doc/tutorials/chapter_9\rohd_vf_example
```

Comment:
is it possible this slash being backwards caused it to still fail?

Reply:
Fixed it! 😀